frontend/src/app/game/pages/outro/outro.component.ts

```diff
@@ -185,8 +190,9 @@ export class OutroComponent {
 
   get i18nData() {
     const stats = this.stats;
+    const detectionRate = this.gameService.game.simulation.params.detectionRate.mean;
```

Comment:
Is always `this.gameService.game` defined at this place?
... and all other lower embedded levels?

Reply:
The whole call is wrapped in `<cvd-col *ngIf="stats">` and the whole calculation doesn't work without initialized game anyways. So yes, it will be defined though I am not sure if that `*ngIf="stats"` should be done differently (use observers?). If there is a problem it was there before this patch.